source/sdk/math.ooc

```diff
@@ -2,6 +2,7 @@ include math
 use math
 
 FLT_EPSILON: extern Float
+DBL_EPSILON: extern Double
```

Comment:
I guess we could, but the way they're meant to be accessed is through the `epsilon` property of `Double` or `Float`.